src/Symfony/Component/Finder/Tests/Comparator/ComparatorTest.php

```diff
@@ -26,14 +32,22 @@ public function testGetSetOperator()
             $this->assertInstanceOf(\InvalidArgumentException::class, $e, '->setOperator() throws an \InvalidArgumentException if the operator is not valid.');
         }
 
-        $comparator = new Comparator();
+        $comparator = new Comparator('some target');
```

Comment:
We have 2 tests in one here. Those should be split (which will also allow using `expectException` to test the exception) instead of this old way of writing tests that does not follow the best practices.

And the test about an invalid operator should also be added for the constructor.

Reply:
#42164